pandas/tests/extension/base/getitem.py

```diff
@@ -245,7 +245,9 @@ def test_take_non_na_fill_value(self, data_missing):
         fill_value = data_missing[1]  # valid
         na = data_missing[0]
 
-        array = data_missing._from_sequence([na, fill_value, na])
+        array = data_missing._from_sequence(
+            [na, fill_value, na], dtype=data_missing.dtype
+        )
```

Comment:
this sort of changes the interface.  do we want authors to handle this on their own?

Reply:
How does it change the interface?

We are reducing coverage of `_from_sequence` inferring the dtype from an untyped list. We could restore that if desired (and probably skip for problematic arrays).

Reply:
> We are reducing coverage of _from_sequence inferring the dtype from an untyped list.

Yah, i guess that is a better description than "changes the interface"